app/models/invite_redeemer.rb

```diff
@@ -36,6 +36,10 @@ def self.create_user_from_invite(invite, username, name, password=nil)
       user.password = password
     end
 
+    user.approved = true
+    user.approved_by_id = invite.invited_by_id
+    user.approved_at = Time.now
```

Comment:
Yes, good point!

Maybe we should only check for staff if "Staff must approve..." is checked?